### ISSUE TITLE
Uninstall script in 0.9.0 has illegal newline.

### ISSUE THREAD

**Issue by fmehta** [FunctionalCorrectness/VS-Code-Language-Server-for-Dafny#1]
Hi

It looks like the uninstall script in 0.9.0 does not work under MacOSx due to a wrong new-line Character.

```
bash: /Users/fm/.vscode/extensions/FunctionalCorrectness.dafny-vscode-0.9.0/scripts/osx/uninstall.sh: /bin/bash^M: bad interpreter: No such file or directory
```

This error seems to crop up again and again. Would there be a way to eliminate it systematically? e.g. A script that does this conversion for the appropriate OS before an install?

Best regards,
Farhad Mehta

**Comment by markusschaden** [comment]
Hi Farhad

Thanks for letting us know.,
This kind of bugs will be gone, end of this week, as soon as we have the native downloading and extracting 
implemented. 

Cheers
Markus

**Comment by fmehta** [comment]
Great.
Thanks,
Farhad

On 10 May 2017, at 10:45, Markus Schaden <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi Farhad

Thanks for letting us know.,
This kind of bugs will be gone, end of this week, as soon as we have the native downloading and extracting
implemented.

Cheers
Markus

**Comment by markusschaden** [comment]
Fixed with 0.10.0 release
